src/platforms/elixir/usage/index.mdx

```diff
@@ -82,8 +82,8 @@ user: %{
 
 ## Breadcrumbs
 
-Sentry supports capturing breadcrumbs – events that happened prior to an issue. We need to be careful because breadcrumbs are per-process. If a process dies it might lose its context.
+Sentry supports capturing breadcrumbs – events that happened prior to an issue. We need to be careful because breadcrumbs are per-process. If a process dies it might lose its context. Breadcrumbs must be in the format described in the [Breadcrumbs](https://docs.sentry.io/product/issues/issue-details/breadcrumbs/#breadcrumbs-display) docs, or else the library will fail to submit correct breadcrumbs.
```

Comment:
```suggestion
Sentry supports capturing breadcrumbs – events that happened prior to an issue. We need to be careful because breadcrumbs are per-process. If a process dies it might lose its context. Breadcrumbs must be in the format described in the [Breadcrumbs](/product/issues/issue-details/breadcrumbs/#breadcrumbs-display) docs, or else the library will fail to submit correct breadcrumbs.
```